src/HypothesisTests.jl

```diff
@@ -30,13 +30,15 @@ using Combinatorics: combinations, permutations
 using Rmath: pwilcox, psignrank
 using Printf: @printf
 
-import StatsAPI
+using StatsAPI: StatsAPI
```

Comment:
Why change this? The current code ensures we have to import any used functions explicitly, which avoids potential issues if StatsAPI exports new functions.
```suggestion
import StatsAPI
```
Or maybe even drop that line entirely:
```suggestion
```

Reply:
I amended the commit to HypothesisTests.jl to minimize changes, in particular I won't touch this line ;)

(all of those changes were auto-formatting)